plume-models/src/users.rs

```diff
@@ -320,13 +320,43 @@ impl User {
             .load::<User>(conn)
             .map_err(Error::from)
     }
-
     pub fn outbox(&self, conn: &Connection) -> Result<ActivityStream<OrderedCollection>> {
-        let acts = self.get_activities(conn)?;
-        let n_acts = acts.len();
         let mut coll = OrderedCollection::default();
+        let first = ap_url(&format!("{}?page=1", &self.outbox_url));
+        let last = ap_url(&format!(
+            "{}?page={}",
+            &self.outbox_url,
+            self.get_activities_count(&conn) / 12
```

Comment:
12 is hard coded. for now.?

Reply:
Oh, we didn't know about the constant elsewhere... But it's in the routes code rather than the models code.

We'll move it over then.